File: JAVA/Curso.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poo1_miercoles;

/**
 *
 * @author Carmen Huanca
 */
public class Curso {
    private String nivel,nomCurso,paralelo;
    private int nroEstudiantes;
    private String nom[]=new String[50];
    private String pat[]=new String[50];
    private String mat[]=new String[50];
    
    public void leer()
    {
        nivel=Leer.dato();
        nomCurso=Leer.dato();
        paralelo=Leer.dato();
        nroEstudiantes=Leer.datoInt();
        for(int i=1;i<=nroEstudiantes;i++)
        {
            nom[i]=Leer.dato();
            pat[i]=Leer.dato();
            mat[i]=Leer.dato();
        }
    }
    public void mostrar()
    {
        System.out.println(nivel+" "+nomCurso+" "+paralelo);
        for(int i=1;i<=nroEstudiantes;i++)
            System.out.println(nom[i]+" "+pat[i]+" "+mat[i]);
    }
    
}

File: C#/Curso.cs
/*
 * Created by SharpDevelop.
 * User: Carmen Huanca
 * Date: 8/24/2022
 * Time: 11:34 AM
 * 
 * To change this template use Tools | Options | Coding | Edit Standard Headers.
 */
using System;

namespace POO1_Miercoles
{
	/// <summary>
	/// Description of Curso.
	/// </summary>
	public class Curso
	{
		private String nivel,nomCurso,paralelo;
	    private int nroEstudiantes;
	    private String[,] datos=new String[50,4];
	   
	    
	    public void leer()
	    {
	    	nivel=Console.ReadLine();
	        nomCurso=Console.ReadLine();
	        paralelo=Console.ReadLine();
	        nroEstudiantes=int.Parse(Console.ReadLine());
	        for(int i=1;i<=nroEstudiantes;i++)
	        {
	            datos[i,1]=Console.ReadLine();
	            datos[i,2]=Console.ReadLine();
	            datos[i,3]=Console.ReadLine();
	        }
	    }
	    public void mostrar()
	    {
	       Console.WriteLine(nivel+" "+nomCurso+" "+paralelo);
	        for(int i=1;i<=nroEstudiantes;i++)
	            Console.WriteLine(datos[i,1]+" "+datos[i,2]+" "+datos[i,3]);
	    }
	}
}
